cartography/intel/aws/lambda_function.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_lambda_data(boto3_session, region):
+    """
+    Create an Lambda boto3 client and grab all the lambda functions.
+    """
+    client = boto3_session.client('lambda', region_name=region)
+    paginator = client.get_paginator('list_functions')
+    lambda_functions = []
+    for page in paginator.paginate():
+        for each_function in page['Functions']:
+            lambda_functions.append(each_function)
+    return ({'AWSLambda': lambda_functions})
```

Comment:
With the decorator added, the function is expected to return just a list, so this line would become:
`return lambda_functions`